Documentation/Readme_edihistory.html

```diff
@@ -21,8 +21,7 @@
 
 -->
 
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
 <head>
```

Comment:
Hi @tywrenn 

So, I guess I should remove this, right?
```
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
```

Reply:
If you did your homework you can do that lol